lib/serre.gi

```diff
@@ -394,7 +399,11 @@ InstallMethod( REPN_ComputeUsingSerre, "for linear reps", [ IsGroupHomomorphism
     end;
 
     parallel := ValueOption("parallel");
-
+    if IsBoundGlobal("ParListByFork") then
+      ParListByFork := ValueGlobal("ParListByFork");
+    else
+      parallel := fail;
+    fi;
```

Comment:
Aha, the code below already has a check for `IO`, and if `parallel <> fail` and `io` is missing it raises an error. So I guess we should respect that here, and instead of setting `parallel := `fail` as I suggested we should have `elif parallel <> fail then Error("...");` ?